src/components/About.jsx

```diff
@@ -1,117 +1,219 @@
-
+import Aos from 'aos';
+import 'aos/dist/aos.css';
 //icons
-import instagramLogo from './assets/images/instagram.svg'
-import whatsappLogo from './assets/images/whatsapp.svg'
-import githubLogo from './assets/images/github.svg'
-
-import aboutSvg from './assets/images/about.svg'
+import instagramLogo from './assets/images/instagram.svg';
+import whatsappLogo from './assets/images/whatsapp.svg';
+import githubLogo from './assets/images/github.svg';
 
-import {device} from '../utils/screenSizes'
+import aboutSvg from './assets/images/about.svg';
 
-const Wrapper =  styled.div`
-    
-    display:flex;
-    flex-direction: column;
+import { device } from '../utils/screenSizes';
 
+import { Jumbotron, Container, Card, Button, Row, Col } from 'react-bootstrap';
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
 `;
 
 const BackgroundImage = styled.img`
-
-
-   @media ${device.mobileS} {
-       align-self:center;
-       max-width: 70%;
+  @media ${device.mobileS} {
+    align-self: center;
+    max-width: 70%;
     flex-direction: column;
-}
-  @media ${device.laptop} { 
-      max-width: 30%;
+    postion: relative;
+  }
+  @media ${device.laptop} {
+    max-width: 30%;
     flex-direction: row;
     margin-right: 2em;
   }
 `;
 
 const Content = styled.div`
-
-    align-self: center;
-    display:flex;
-    margin: 5em 0;
-    width: 80%;
-
-    margin: auto auto;
-    box-shadow: 2px 2px 5px rgba(0,0,0,0.5);
-    padding: 2em;
-   
-    
-      @media ${device.mobileS} {
-        flex-direction: column;
-
-      }
-      @media ${device.laptop} { 
-        flex-direction: row;
-      }
-  
+  background-image: url(https://www.xmple.com/wallpaper/linear-white-gradient-highlight-blue-1920x1080-c2-ffffff-e0ffff-l-50-a-300-f-21.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+
+  display: flex;
+  width: 100%;
+  margin: 0px;
+  // box-shadow: 2px 2px 5px rgba(0, 0, 0, 0.5);
+  padding: 4em;
+  @media ${device.mobileS} {
+    flex-direction: column;
+    margin: 0px;
+    padding: 2em 20px;
+    text-align: left;
+  }
+  @media ${device.laptop} {
+    flex-direction: row;
+    padding: 50px;
+    margin: 0px;
+  }
 `;
 
 const Text = styled.div`
-    font-size: 20px;
-    margin-right: 1em;
-    
-    @media ${device.mobileS} {
-        flex-direction: column;
-
-      }
-      @media ${device.laptop} { 
-        width: 60%;
-        flex-direction: row;
-      }
+  margin-right: 1em;
+  font-family: 'Montserrat', sans-serif;
+  font-weight: light;
+  @media ${device.mobileS} {
+    flex-direction: column;
+    font-size: 15px;
+  }
+  @media ${device.laptop} {
+    width: 60%;
+    flex-direction: row;
+    font-size: 18px;
+  }
 `;
 const Logos = styled.div`
-    display:flex;
-    
-    justify-content: stretch;
-    align-self: center;
-    border-top: 1px solid rgba(0,0,0,0.3);
-    
-   
-    padding: 2em;
-    /* height: 50em; */
+  display: flex;
```

Comment:
No Need of these properties,
width , display , margin. You are mixing properties unnecessarily.